script/message-spec.py

```diff
@@ -12,10 +12,14 @@
         description="Generate an AsyncAPI message spec from an HMPPS domain event message."
     )
     parser.add_argument("event", type=str, help="An HMPPS domain event type")
+    parser.add_argument("-j", "--json", action='store_true')
     args = parser.parse_args()
 
     # Grab a single application insights domain event message
     message = lib.application_insights.single_domain_event(args.event)
 
-    # Write an ASyncAPI YAML message based on the event
-    print(lib.asyncapi.create_asyncapi_yaml(message))
+    if args.json:
+        print(message)
```

Comment:
If we serialise the JSON it can be used directly with `check-jsonschema`

```suggestion
import json
...
        print(json.dumps(message))
```